delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1041,21 +1040,32 @@ def updateServiceInDB(service_id_in_db, devApiID, region, statusval) {
 def loadEventsModule() {
 }
 
+	
 def loadServiceInfo() {
 	def token = setCredentials()
 	def url = "$g_base_url/platform/services?domain=$domain&service=$service&isAdmin=true"
 	def curlOutput = sh (script: "curl -H \"Content-Type: application/json\" \
-			-H \"Authorization: $token\" \"$url\" | $jq -r '.data[0] | [.created_by] | join(\",\")'", returnStdout: true)
-	def values = curlOutput.split(',')
-
+			-H \"Authorization: $token\" \"$url\"", returnStdout: true)
+  	echo "CURL o/p: $curlOutput"
+  	
+	def jsonParser = new groovy.json.JsonSlurper()
+	def resultJson = jsonParser.parseText(curlOutput)
+  	echo "Resultjson: $resultJson"
+	def createby = ''
+  	def pcreated = resultJson.data[0].created_by
+  	echo "resultjson createby: $pcreated"
+	if(resultJson != null && resultJson.data != null && resultJson.data[0].created_by != null) {
+			createby = resultJson.data[0].created_by
+		}
+	def values = createby.split(',')
```

Comment:
This line should be in above bracket so you can be sure createdby is not null before attempting to split

Reply:
Code Changed as suggested